scoreboard/rest.py

```diff
@@ -337,6 +354,7 @@ def post(self, email):
 
 api.add_resource(UserList, '/api/users')
 api.add_resource(User, '/api/users/<int:user_id>')
+api.add_resource(TeamChange, '/api/teams/change')
```

Comment:
I was torn myself. On the one hand, it seems a little jury-rigged, but on the other hand it allows us to use the same schema without disrupting anything